common/functions-v2/src/scalars/string.rs

```diff
@@ -193,6 +193,139 @@ pub fn register(registry: &mut FunctionRegistry) {
         },
         |val| val.first().cloned().unwrap_or_default(),
     );
+
+    // Trim functions
+    registry.register_passthrough_nullable_1_arg::<StringType, StringType, _, _>(
+        "ltrim",
+        FunctionProperty::default(),
+        |_| None,
+        vectorize_string_to_string(
+            |_| 0,
```

Comment:
If is it better to reserve some space for output?

Reply:
I think we could reserve same length as the input.